controllers/cards.js

```diff
@@ -1,26 +1,38 @@
 const Card = require('../models/card');
 
+const { errorSend } = require('../config.js');
+
 module.exports.getCards = (req, res) => {
   Card.find({})
-    .then((card) => res.send({ data: card }))
-    .catch(() => res.status(500).send({ message: 'Произошла ошибка на сервере' }));
+    .then((card) => {
+      if (card.length === 0) {
+        return res.status(404).send({ message: 'Данных нет!' });
+      }
+      return res.send({ data: card });
+    })
+    .catch(() => errorSend(res));
 };
 
 module.exports.createCard = (req, res) => {
   const owner = req.user._id;
   const { name, link } = req.body;
   Card.create({ name, link, owner })
     .then((card) => res.send({ data: card }))
-    .catch(() => res.status(500).send({ message: 'Произошла ошибка на сервере' }));
+    .catch(() => errorSend(res));
 };
 
 module.exports.deleteCard = (req, res) => {
-  Card.findByIdAndRemove(req.params.cardId)
+  const ownerId = req.user._id;
+  const { cardId } = req.params.cardId;
```

Comment:
Нужно исправить: ошибка при удалении карточки.
```suggestion
  const { cardId } = req.params;
```
или
```suggestion
  const cardId = req.params.cardId;
```

Reply:
Исправил